unit-tests/unit-tests-post-processing.cpp

```diff
@@ -193,7 +193,7 @@ const std::vector< std::pair<std::string, std::string>> ppf_test_cases = {
 
 // The test is intended to check the results of filters applied on a sequence of frames, specifically the temporal filter
 // that preserves an internal state. The test utilizes rosbag recordings
-TEST_CASE("Post-Processing Filters sequence validation", "[software-device][post-processing-filters]")
+TEST_CASE("Post-Processing Filters sequence validation", "[software-device-disabled][post-processing-filters]")
```

Comment:
We will remove all at the last test that is converted?

Reply:
We can :)